From 597f61f3a2da1f92e8bea09f3daf5c7da16133a2 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Fri, 2 Sep 2016 09:29:35 -0700
Subject: [PATCH] this is a test change Reviewed by: David Pacheco
 <dap@joyent.com> Approved by: David Pacheco <dap@joyent.com>

---
 README.md | 1 -
 1 file changed, 1 deletion(-)

diff --git a/README.md b/README.md
index f6da383..75cf6c7 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,3 @@
 # gerrit-test
 
 This is a dummy repository for verifying cr.joyent.us behavior.
-Test update.
-- 
2.21.0

